source/crud/crud.rst

```diff
@@ -1573,6 +1605,7 @@ Q: Where is ``useCursor`` in AggregateOptions?
 Changes
 =======
 
+* 2017-10-06: Split UpdateOptions and ReplaceOptions.
```

Comment:
Note: FindOneAndUpdateOptions and FindOneAndReplaceOptions were already split and only FindOneAndUpdateOptions received the new `arrayFilters` option, so those classes weren't a concern.